package.json

```diff
@@ -27,10 +27,17 @@
     "url": "https://github.com/strongloop/loopback-soap/issues"
   },
   "devDependencies": {
+    "chai": "^3.2.0",
     "eslint": "^3.12.2",
-    "eslint-config-loopback": "^7.0.0"
+    "eslint-config-loopback": "^8.0.0",
+    "loopback": "^3.0.0",
```

Comment:
This package doesn't actually use loopback, as far as I can tell. I was able to remove this line and `git clean -fxd && npm install && npm test` still passed.